client/PeopleSelection.jsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react';
 
 class PeopleSelection extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {};
+  createSelections() {
```

Comment:
Did you mean to delete the constructor but still keep this as a class?

Reply:
thanks for this suggestion, i've changed this now to be a functional component